integ-test/src/test/java/org/opensearch/sql/sql/NestedIT.java

```diff
@@ -87,8 +87,7 @@ public void nested_function_with_arrays_in_an_aggregate_function_in_select_test(
     verifyDataRows(result, rows(19));
   }
 
-  // TODO not currently supported by legacy, should we add implementation in AstBuilder?
-  @Disabled
+  @Test
   public void nested_function_in_a_function_in_select_test() {
```

Comment:
do we have a similar test in WHERE?  Maybe in ORDER BY too?

Reply:
I'll add one for WHERE, ORDER BY is out of scope and not supported with this PR.